mkt/websites/models.py

```diff
@@ -116,6 +118,17 @@ def get_icon_url(self, size):
     def get_url_path(self):
         return reverse('website.detail', kwargs={'pk': self.pk})
 
+    def get_regions(self, sort_by='slug'):
```

Comment:
I wouldn't call this `get_regions`: this is confusing since the method already exists on webapps, but unlike webapps websites are available in all regions anyway